sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -55,7 +55,7 @@ public class EventProcessorClient
         private Func<ProcessErrorEventArgs, Task> _processErrorAsync;
 
         /// <summary>Indicates whether or not this event processor is currently running.  Used only for mocking purposes.</summary>
-        private bool? _isRunningMock;
+        private bool? _isRunningOverride;
```

Comment:
Yeah.... that's spot-on clear for what it does.  nice.

Reply:
I wanted to include a `remark` like you suggested, but I wasn't sure if you wanted me to add it here or in the `IsRunning` property.

Reply:
I think that what you've got here is clear enough.   Since it's only intended to be overridden in mocking scenarios, I don't think there's value in documenting it at the properly level; the majority of our users won't find value there.